apps/sheriff-webservices/package.json

```diff
@@ -8,16 +8,15 @@
     "build": "tsc",
     "typecheck": "tsc --noEmit",
     "lint": "eslint . --max-warnings=0 --cache --cache-location=node_modules/.cache/.eslintcache",
-    "typesync": "typesync --dry=fail",
     "serve": "node ./dist/index.js",
     "clean": "rm -rf .turbo dist node_modules/.cache"
   },
   "devDependencies": {
     "@sherifforg/types": "workspace:*",
     "@tsconfig/node-lts": "^18.12.5",
-    "@types/eslint-plugin-jsx-a11y": "^6.9.0",
-    "@types/eslint__js": "^8.42.3",
-    "@types/lodash": "^4.17.6",
+    "@types/eslint-plugin-jsx-a11y": "~6.9.0",
+    "@types/eslint__js": "~8.42.3",
```

Comment:
oh damn, this is old stuff, we don't need this anymore

Reply:
They're dropped in master.